modules/codegen/src/main/scala/com/twilio/guardrail/protocol/terms/client/ClientTerms.scala

```diff
@@ -10,31 +10,25 @@ import com.twilio.guardrail.terms.{ RouteMeta, SecurityScheme }
 import com.twilio.guardrail.{ RenderedClientOperation, StaticDefns, StrictProtocolElems, SupportDefinition }
 import java.net.URI
 
-class ClientTerms[L <: LA, F[_]](implicit I: InjectK[ClientTerm[L, ?], F]) {
+abstract class ClientTerms[L <: LA, F[_]] {
```

Comment:
Any particular reason for using abstract classes rather than traits?

Reply:
I thought it would be fewer moving pieces to go from class to abstract class, but I guess it does prevent implementing multiple different of these interfaces via multiple mixin, even if we don't do it here, so I can change it.